rim/templates/rim/client_list.html

```diff
@@ -0,0 +1,39 @@
+{% extends "rim/base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block content %}
+  <div class='grid-x grid-padding-x'>
+    <div class='small-7 large-7 cell'>
+      <h1>{% trans 'Client List' %}</h1>
+    </div>
+    <div class ='small-5 large-5 cell'>
+      <form method=GET class='clientform' action="{% url 'client_list' %}">
+        <input type='text' placeholder='search name or M number' name='search' value='{{searchterm}}'>
```

Comment:
The placeholder text needs a [`{% trans %}`](https://docs.djangoproject.com/en/2.1/topics/i18n/translation/#trans-template-tag) tag around it (also maybe use sentence case and "M-number" instead of "M number")

Reply:
fixed